Section_I/law4.tex

```diff
@@ -285,25 +276,37 @@
 
 \headlinebox
 
-In the event of any infringement of this Law:
+In a virtual competition, the equipment of the players is checked by the Technical Committee prior to the tournament.
+
+\bigskip
+
+In the event of any infringement of this Law for a virtual tournament:
+
+\begin{itemize}
+\item The Technical Committee notifies the team in advance of the tournament about the violations and allows them to correct the equipment of the players.
+\item In case no valid robot model has been provided prior to the tournament, the team is excluded from participation.
+\end{itemize}
+
+In the event of any infringement of this Law occurring during a physical tournament:
 
 \begin{itemize}
 \item play need not be stopped
 \item the player at fault is instructed by the referee to leave the field of play to correct his equipment
 \item the player leaves the field of play when the ball next ceases to be in play, unless he has already corrected his equipment
 \item any player required to leave the field of play to correct his equipment must not re-enter without the referee's permission
-\item the referee checks that the player's equipment is correct before allowing him to re-enter the field of play 
+\item the referee checks that the player's equipment is correct before allowing him to re-enter the field of play
 \item the player is only allowed to re-enter the field of play before the respective penalty time is over \greyed{ (replaces: when the ball is out of play)}
 \end{itemize}
 
 \bigskip
 
-A player who has been required to leave the field of play because of an infringement of this Law and who re-enters the field of play without the referee's permission must be cautioned.
+A player who has been required to leave the field of play during a physical game because of an infringement of this Law and who re-enters the field of play without the referee's permission must be cautioned.
```

Comment:
Because robots in the virtual competition cannot be uncompliant with the rules in this section as they are tested by the TC beforehand and the simulator doesn't allow for parts falling off, I believe. So the robots that start playing the game will be in the same state throughout the game, so this never applies.